src/org/sosy_lab/java_smt/api/ArrayFormulaManager.java

```diff
@@ -61,6 +61,30 @@ <TI extends Formula, TE extends Formula> ArrayFormula<TI, TE> store(
   <TI extends Formula, TE extends Formula> ArrayFormula<TI, TE> makeArray(
       String pName, ArrayFormulaType<TI, TE> type);
 
+  /** Create a new array constant with uninitialized values. */
+  <TI extends Formula, TE extends Formula, FTI extends FormulaType<TI>, FTE extends FormulaType<TE>>
```

Comment:
I do not yet see the benefit of having uninitialized unnamed arrays in the API.
Please conosider the following usages:
- Create such an array and then use the Java instance twice. Should these be two distinct arrays or the same array? (There is no difference to having a array variable.)
- Create two distinct uninitialized arrays. Those arrays are distinct and can hold distinct or equal elements. (There is no difference to having two array variables.)

Overall, the user can simply choose a unique name and create an array directly.

Reply:
That's probably right.